src/authentication.ts

```diff
@@ -0,0 +1,29 @@
+import { Request } from "express";
+
+/**
+ * Simple authentication function for tsoa
+ * In a real application, this would validate JWT tokens, API keys, etc.
+ */
+export function expressAuthentication(
```

Comment:
move all tsoa files (auth + ios) to separate 'openapi' foleder
and all generated files there as well